common/__init__.py

```diff
@@ -41,6 +44,26 @@ def prepare_server_properties(properties: List[str]) -> List[str]:
     none_empty = list(filter(None, stripped))
     return none_empty
 
+def deep_get(dictionary: Dict[str, Any], *keys):
+    return reduce(lambda d, key: d.get(key) if d else None, keys, dictionary)
+
+PHANTOM_HTML = """
+<style>div.phantom {{font-style: italic; color: #00ff73}}</style>
+<div class='phantom'>{}</div>"""
```

Comment:
You may want to use something different from the hard-coded color #00ff73.

You could make it color-scheme-independent by selecting the color of the user's comment color perhaps:

```python
color = view.style_for_scope("comment")["foreground"]
```

For my particular color-scheme, it prints:

```
>>> view.style_for_scope("comment")["foreground"]
'#a6acb9'
```

There are many more standard scopes that should be present in a color scheme. I'll refer to the "minimal scope coverage" at the bottom of this guide: http://www.sublimetext.com/docs/3/scope_naming.html

Reply:
I wasn't aware of that. I followed you suggestion :+1: